### DISCUSSION TITLE
SSR without any client-side JavaScript

### DISCUSSION THREAD

**Discussion by omurad** [nuxt/nuxt#10213] [Questions (Nuxt 2)] [answered]
I'd like to write an app without any client-side JS and using NuxtJS seems like a great, scalable option compared to something like HandlebarsJS.

Is there an official way to not include any JS in the rendered pages?

I have seen some old threads demonstrating hacky ways of achieving this, but I am wondering if an official solution has come out since?

**Comment by danielroe** [comment, marked as answer]
Yes, this can be done straightforwardly, since 2.8.0.

```js
export default {
  render: {
    injectScripts: process.env.NODE_ENV !== 'production'
  }
}
```

More info: https://nuxtjs.org/docs/configuration-glossary/configuration-render#injectscripts

**Reply by omurad** [reply]
Thank you!

I'm surprised I couldn't find this option after many google searches.

**Reply by Susccy** [reply]
How does this setting behave with "static" JS such as for animations and similar? Does that get removed too?
